vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -261,30 +282,30 @@ namespace react { namespace uwp {
       auto scheme = uri.SchemeName();
       needsDownload = scheme == L"http" || scheme == L"https";
 
-      EmitImageEvent(instance, image, "topLoadStart", sources[0]);
+      EmitImageEvent(instance, canvas, "topLoadStart", sources[0]);
       if (needsDownload)
       {
-        // FUTURE: This should get a weak_ptr from instance
-        // fix when the win32 instance merge happens
-        DownloadImageAsync(image, sources[0], instance);
+        auto reactImage{ canvas.as<ReactImage>() };
+        reactImage->SourceUri(uri);
```

Comment:
Where is the download code for this path?  DownloadImageAsync path is necessary for for http/https to support when people set method/headers  (we could condition our needsSpecialDownload more based on that and if they are requesting load status events) but to keep currently behavior we need to be running that path.  In our people card code auth tokens are specified to download images.

Reply:
Changed DownloadImageAsync to GetImageStreamAsync and moved it to ReactImage.

GetImageStreamAsync is now used by ReactImage (when you set the Source) and also by ImageViewManagerModule (through GetImageSizeAsync which is called by getSize)